backend/sirius-web-spring-collaborative/pom.xml

```diff
@@ -73,6 +73,10 @@
 			<artifactId>sirius-web-representations</artifactId>
 			<version>2021.12.14</version>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
```

Comment:
Could you extract in a specific commit:
- this change in the dependencies of this project
- The introduction of the DelegatingRequestContextExecutorService
- The introduction of the DelegatingRequestContextCallable
- The introduction of the DelegatingRequestContextRunnable
- The change of the EditingContextEventProcessor

and update the CHANGELOG to detail this new feature.

Reply:
Consumer of Sirius Components need to understand that starting from this change, they will have to provide the `RequestAttributes` in the `RequestContextHolder` to make the `EditingContextEventProcessor` work. It should be very easy for those in a Spring environment but it is not free for those outside of a Spring environment.